skymap_scanner/server/start_scan.py

```diff
@@ -691,16 +691,20 @@ def _nside_and_pixelextension(val: str) -> Tuple[int, int]:
         event_contents = fetch_event_contents_from_file(args.event_file)
 
     # get inputs (load event_id + state_dict cache)
+    # NOTE: if pulses_name is None, it is automatically determined from the realtime format version
+    # leave argument to support passing an arbitrary pulses_name in the future
+    LOGGER.info("Extracting event...")
     event_metadata, state_dict = extract_json_message.extract_json_message(
         event_contents,
         reco_algo=args.reco_algo,
         is_real_event=args.real_event,
         cache_dir=str(args.cache_dir),
         GCD_dir=str(args.gcd_dir),
-        pulsesName=cfg.INPUT_PULSES_NAME,
+        pulses_name=None,
```

Comment:
When would this not be `None`? I'm concerned that this may be configuration creep, ie supporting more options than needed.

Reply:
@tianluyuan at some point mentioned that it would be nice to be able to optionally take a different pulse series as input, on-demand. So either we establish a plan to support it, or we drop the argument.

Note that such a feature can be a bit tricky to implement, given that for a given pulse series, the reco algos may try to look up associated entries in the frame (for example the corresponding `TimeRange`).

Reply:
Maybe we can just remove the argument for now. If that enhancement makes it in it can be added back.